.travis.yml

```diff
@@ -1,6 +1,7 @@
+dist: bionic
 language: python
-
-sudo: required
+python:
+  - "3.8.0"
```

Comment:
Currently, the default python build version is `3.6.7`
https://travis-ci.com/chenrui333/docker-selenium/jobs/247355547#L179